build/python-check.sh.in

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-PEP8="@PEP8@"
-PYFLAKES="@PYFLAKES@"
+PEP8="pycodestyle"
+PYFLAKES="pyflakes"
```

Comment:
Any reason for using pycodestyle and pyflakes?
I think other projects use flake8 for example, which does both at once.